qiita_db/metadata_template.py

```diff
@@ -135,14 +135,14 @@ def _as_python_types(metadata_map, headers):
     return values
 
 
-def _prefix_sample_names_with_id(md_template, study):
+def _prefix_sample_names_with_id(md_template, study_id):
```

Comment:
Talked with @josenavas and we agreed that it should stay with the ID, not the study object itself.